src/gd_kotlin.cpp

```diff
@@ -129,6 +138,13 @@ void GDKotlin::init() {
             if (split_jvm_debug_argument(cmd_arg, result) == OK) {
                 gc_thread_period_interval = result.to_int64();
             }
+        } else if (cmd_arg.find("--jvm-to-engine-shared-buffer-size") >= 0) {
+            String result;
+            if (split_jvm_debug_argument(cmd_arg, result) == OK) {
+                jvm_to_engine_shared_buffer_size = result.to_int();
+                print_line(vformat("Warning ! Buffer capacity was changed to %s, this is not a recommended practice",
```

Comment:
Shouldn't this be a warning?

Reply:
Maybe also add a todo so the warning links to the documentation once we have one